backtrace-library/src/main/java/backtraceio/library/interfaces/Database.java

```diff
@@ -2,14 +2,13 @@
 
 import java.util.Map;
 
-import backtraceio.library.BacktraceClient;
 import backtraceio.library.BacktraceCredentials;
 import backtraceio.library.base.BacktraceBase;
 import backtraceio.library.models.database.BacktraceDatabaseRecord;
 import backtraceio.library.models.database.BacktraceDatabaseSettings;
 import backtraceio.library.models.json.BacktraceReport;
 
-public interface Database {
+public interface Database extends Breadcrumbs {
```

Comment:
I would say that this interface (Database) should be separate from Breadcrumbs. Of course Backtrace Database can implements both interfaces

Reply:
If I do this I have to cast every time I want `BacktraceBase` to call `database.addBreadcrumb`. Is this okay?

Reply:
Which reminds me - we might want to expose breadcrumbs object from database instead of exposing breadcrumbs method in backtrace database - with this approach we will remove a lot of unnecessary code. Let's sync up on this today with my simple proposal to remove this interface!

Reply:
That is an excellent proposal Konrad, I will do that

Reply:
Done, it looks much better now. Thank you both!